src/app/app.component.html

```diff
@@ -0,0 +1,8 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-info py-3" >&nbsp;  &nbsp;
+     <a routerLink="home" class="text-light" >Home</a> &nbsp;  &nbsp;
+  <a routerLink="header" class="text-light" >Header</a> &nbsp;  &nbsp;
+  <a routerLink="footer"  class="text-light">Footer</a> 
+</nav>
```

Comment:
Move to component

Reply:
which component ?? I moved to the header component. is that right?